src/components/SideNavBar/SideNavBar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export const GenerateHeader = ({ icon, title }) => {
```

Comment:
We can take this function out in a separate file

Reply:
I would be removing this file as there is change in design